mappings/net/minecraft/client/render/block/FluidRenderer.mapping

```diff
@@ -17,10 +17,10 @@ CLASS net/minecraft/class_775 net/minecraft/client/render/block/FluidRenderer
 		ARG 1 world
 		ARG 2 pos
 	METHOD method_3344 isSideCovered (Lnet/minecraft/class_1922;Lnet/minecraft/class_2338;Lnet/minecraft/class_2350;F)Z
-		ARG 0 world
+		ARG 0 view
```

Comment:
Player suggests to keep these parameters world even though they may be of a XxxView type.

Reply:
Yeaaaah. I can accept that. I'll do it as another PR.

Reply:
@liach #949 is the number.